.gitmodules

```diff
@@ -1,3 +1,19 @@
 [submodule "external/monero-project"]
 	path = external/monero-project
-	url = https://github.com/woodser/monero
+	url = https://github.com/nsec1/monero.git
+	branch = fixIncludePaths
+[submodule "external/boost"]
```

Comment:
Thinking about this further, I'm wondering if we should checkout these submodules as part of the monero-cpp project, to be generally available within that project, or if they're only necessary in the way we're building monero-java?

monero-ts, for example, does not require these submodules to be checked out, and I wouldn't want them to interfere with the build in that project.

But I could understand the argument to make these generally available in monero-cpp for other build setups.

Curious to get your thoughts?

Reply:
I followed your suggestion: https://github.com/woodser/monero-java/pull/105#issuecomment-2295302158
and I agreed :)
We can pass a env var to on/off submodules build